### PR TITLE
📝 Add Pull Request Template

### PR DESCRIPTION
**Pull request by mistryrn**
* Adds `.github/pull_request_template.md` template for PR descriptions

For quick reference, below is what the template will look like in full:

## PR Readiness Checklist

- [ ] "Expected Outcome(s)" in ticket have been met
- [ ] Ticket number included in PR title
- [ ] Connected ticket to PR
- [ ] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [ ] Manual testing completed
- [ ] Builds locally without errors or warnings
- [ ] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [ ] New environment variables added to `.env.schema` files, `README.md`
- [ ] Added copyrights to new files

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## Description of Changes
+
+Describe the changes in your pull request, providing enough context for reviewers. Try to describe the changes *per service*.
+
+Be sure to call out any breaking changes, as well as any special instructions required to run the new code (i.e. New or updated dependencies? `pnpm i`. New migrations to run? `pnpm run migrate-dev`. etc.)
+
+## Services Affected
+
+- [ ] `consent-ui`
```
Comment (anncatton) [inline .github/pull_request_template.md:9]: not a service exactly, but maybe add a `docs` item

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] `consent-api`
+- [ ] `data-mapper`
+- [ ] `consent-das`
+- [ ] `keys-das`
+- [ ] `pi-das`
+- [ ] `phi-das`
+
```
Comment (anncatton) [inline .github/pull_request_template.md:16]: not sure if it should be a separate header + list, but we'll need to reference the `packages` as well. do you think it makes more sense to have an exact list of all the packages (which i imagine will change over time) or just a section for `Packages Affected` and then the PR owner can list which ones have been modified

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## PR Readiness Checklist
+
+- [ ] "Expected Outcome(s)" in ticket have been met
+- [ ] Manual testing completed
+- [ ] Builds locally without errors
```
Comment (anncatton) [inline .github/pull_request_template.md:21]: maybe also add `without warnings` as well, i think that would address any missing linting/formatting

Comment (mistryrn) [reply .github/pull_request_template.md:21]: Good call, will update that!

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+## Description of Changes
+
+Describe the changes in your pull request **per service or package**, providing enough context for reviewers.
+
+Be sure to call out any breaking changes, as well as any special instructions required to run the new code (i.e. New or updated dependencies? `pnpm i`. New migrations to run? `pnpm run migrate-dev`. etc.)
```
Comment (joneubank) [inline .github/pull_request_template.md:5]: If you wrap these instructions in HTML comments then they won't appear in the PR description if someone forgets to delete them.

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+## PR Readiness Checklist
+
+- [ ] "Expected Outcome(s)" in ticket have been met
+- [ ] Manual testing completed
+- [ ] Builds locally without errors or warnings
+- [ ] Tests are updated (if required) and passing
+- [ ] PR feedback has been addressed
+- [ ] PR branch is up-to-date with target branch and there are no merge conflicts
```
Comment (joneubank) [inline .github/pull_request_template.md:14]: Don't really need this check as this will be shown by github

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+- [ ] New environment variables added to `.env.schema` files, `README.md`
```
Comment (joneubank) [inline .github/pull_request_template.md:15]: 👍

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+- [ ] Added copyrights to new files
+- [ ] Connected ticket to PR
```
Comment (joneubank) [inline .github/pull_request_template.md:17]: I might put this higher up the list so people don't miss it. Maybe even first.

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+- [ ] Ticket number included in PR title
+- [ ] Labels added to PR
```
Comment (joneubank) [inline .github/pull_request_template.md:19]: Also put these at top, these are good to have as reminders.

File: .github/pull_request_template.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+## Description of Changes
+
+Describe the changes in your pull request **per service or package**, providing enough context for reviewers.
```
Comment (joneubank) [inline .github/pull_request_template.md:3]: You may want to define the formatting for the **per service or package** description and give an example or template.

File: .github/pull_request_template.md (hunk 2 above)
Comment (samrichca) [reply .github/pull_request_template.md]: apps/services & packages are defined separately in other places, I would separate them here for consistency

File: .github/pull_request_template.md (hunk 8 above)
Comment (samrichca) [inline .github/pull_request_template.md:19]: what kinds of labels?

**Comment by samrichca** [comment]
what should PRs look like for release candidates?

File: .github/pull_request_template.md (hunk 8 above)
Comment (joneubank) [reply .github/pull_request_template.md:19]: Yeah agreed, I just tried to use this template and this bullet point was a head scratcher. If we want labels we should include instructions about what labels should be used and when to use them.

**Review by anncatton — APPROVED** [review]
🎨 🌈 ❤️ 🌈 🎨  thanks @mistryrn !

**Review by joneubank — APPROVED** [review]

**Pull request merged**